docs/docs/reference/anomaly_detection.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 1
+---
+
+# Anomaly Detection
+
+> In data analysis, anomaly detection (also referred to as outlier detection) is generally understood to be the
+> identification of rare items, events or observations which deviate significantly from the majority of the data.
+> Such examples may arouse suspicions of being generated by a different mechanism, or appear inconsistent with the data.
+>
+> -- Wikipedia
+
+re_data supports three types on anomaly detection for monitoring data.
+
+### Z Score
+- In a statistical distribution, Z-score tells you how far is a given data point from the rest of the crowd. Technically speaking, Z-score measures how many standard deviations away a given observation is from the mean   
+- Z Score = $\dfrac{x - mean}{Standard Deviation}$
+- Configuring Z Score:
+```yaml
+# globally
+vars:
+    re_data:anomaly_detector:
+        name: z_score
+        threshold: 3
+
+# configuring z score as anomaly detector per model
+{{
+    config(
+      re_data_monitored=true,
+      re_data_anomaly_detector={'name': 'z_score', 'threshold': 3} }}
+    )
+}}
+
+```
```

Comment:
It would be really good to give some recommendations when each method may work best? Any idea on that based on the articles?